comp110-worksheet-7/DirectoryUtils.cs

```diff
@@ -9,6 +9,32 @@ namespace comp110_worksheet_7
 {
 	public static class DirectoryUtils
 	{
+        public static List<string> Traverse(string filePath, bool includeFiles, bool includeFolders)
```

Comment:
Good idea to define this helper function!

Boolean parameters can sometimes be confusing, especially when there is more than one (you have to look at the function signature to find out what they mean) -- look into using an enum with the `[Flags]` attribute for this kind of situation.